wsgi/odyssy/announcement/templates/announcement/info.html

```diff
@@ -0,0 +1,10 @@
+<body>
+  {% if info %}
+    <h1>{{ info.title }}</h1><br>
+    {% if info.description is not None %}
+      <p>{{ info.description }}</p>
```

Comment:
Just change `info.description` to `info.description|safe`. Refer to #20 more details.